src/scss/customOverrides.scss

```diff
@@ -1,3 +1,12 @@
 @import "scss/constants.scss";
 
 // This file exists for custom styles to deal with environment specific hacks/overrides
+
+.Badges {
+  &__visits {
+    display: none;
+  }
+  &__visit {
+    display: none;
+  }
+}
```

Comment:
@whoaitsaimz What are these styles even targeting? Searching the codebase for `__visit` or `__visits` doesn't even turn anything up for me..?